File: FrontendApp/vaultextestweb/src/components/Employee.js
import './Employee.css';

  function Employee(props) {
    console.log(props);
    return (
     
      <tr className="tablestyle">
        <td className='tableRow'>{props.FirstName}</td>
        <td className='tableRow'>{props.LastName}</td>
        <td className='tableRow'>{props.OrganisationName}</td>
      </tr>
    );
}

export default Employee;

File: VaultextestDemoApp/Models/Employee.cs
using System;
using System.Collections.Generic;

namespace VaultextestDemoApp.Models
{
    public partial class Employee
    {
        public string? OrganisationNumber { get; set; }
        public string? FirstName { get; set; }
        public string? LastName { get; set; }
    }
    public class EmployeeDTO : Employee
    {
        public string? OrganisationName { get; set; }
    }
}
